packages/sui-lint/bin/sui-lint-js.js

```diff
@@ -13,14 +13,16 @@ program
   .option('--staged')
   .option('--fix', 'fix automatically problems with js files')
   .option('--ignore-patterns <ignorePatterns...>', 'Path patterns to ignore for linting')
+  .option('--reporter <reporter>', 'Send results to DD using sui-logger')
```

Comment:
Maybe the message should be more generic than "'Send results to DD using sui-logger", isn't it?